src/client/models/tankMovement.js

```diff
@@ -1,7 +1,12 @@
-import constans from './constans';
-var originalPoints = constans.originalPoints;
+import externalVariables from './externalVariables';
+let originalPoints = externalVariables.originalPoints,
```

Comment:
I'm going to modify **originalPoints** after each damage on battleground.
Should it be **const**